### PR TITLE
Update October release notes

### PR DESCRIPTION
**Pull request by rmshaffer**
Removes line about IQ# debug experience and adds links to closed PRs.

### CODE REVIEW

File: articles/resources/relnotes.md [hunk 1]
```diff
@@ -23,9 +23,10 @@ For update instructions, please refer to the [update guide](xref:microsoft.quant
 
 This release contains the following:
 
-- Improved IQ# debug user experience, to help developers better understand the inner workings of their quantum applications, and more easily identify logic errors. Created visualizations of a particular execution path of a quantum application, as well as the state at a point during the program execution.
 - Implemented a way for resource estimation to produce simultaneously achievable depth and width estimates.
```
Comment (bettinaheim) [inline articles/resources/relnotes.md:26]: If I could add a link with this one:
```suggestion
- Resource estimation now emits simultaneously achievable depth and width estimates in addition to the qubit count, see [here](xref:microsoft.quantum.machines.qc-trace-simulator.intro) for detail.
```

File: articles/resources/relnotes.md [hunk 2]
```diff
@@ -23,8 +23,9 @@ For update instructions, please refer to the [update guide](xref:microsoft.quant
 
 This release contains the following:
 
-- Improved IQ# debug user experience, to help developers better understand the inner workings of their quantum applications, and more easily identify logic errors. Created visualizations of a particular execution path of a quantum application, as well as the state at a point during the program execution.
-- Implemented a way for resource estimation to produce simultaneously achievable depth and width estimates.
+- Resource estimation now emits simultaneously achievable depth and width estimates in addition to the qubit count. See [here](xref:microsoft.quantum.machines.qc-trace-simulator.intro) for details.
```
Comment (rmshaffer) [inline articles/resources/relnotes.md:26]: @bettinaheim I applied your suggestion, but from what I can see, that link doesn't seem to actually discuss this feature directly? Did you mean perhaps this link instead?

```suggestion
- Resource estimation now emits simultaneously achievable depth and width estimates in addition to the qubit count. See [here](xref:microsoft.quantum.machines.resources-estimator) for details.
```

**Review by ricardo-espinoza — APPROVED** [review]

File: articles/resources/relnotes.md (hunk 2 above)
Comment (bettinaheim) [reply articles/resources/relnotes.md:26]: The best link would be to here: https://github.com/MicrosoftDocs/quantum-docs-pr/blob/main/articles/user-guide/machines/resources-estimator.md#metrics-reported

Comment (rmshaffer) [reply articles/resources/relnotes.md:26]: No idea if `#` fragments work in these `xref` links, but let's give it a shot...

Comment (rmshaffer) [reply articles/resources/relnotes.md]: Yes, it worked!

**Pull request merged**